### ISSUE TITLE
It supports batchsize>1？

### ISSUE THREAD

**Issue by gjd2017** [longcw/RoIAlign.pytorch#16]
Hi, dose it support batchsize>1?

**Comment by longcw** [comment]
Yes, it supports batch > 1. You can check the example here: https://github.com/longcw/RoIAlign.pytorch/blob/master/tests/test.py#L114
